ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/EmailTemplateFormModal.tsx

```diff
@@ -0,0 +1,286 @@
+                                    <TextContent>
+                                        <Text
+                                            component={TextVariants.p}
+                                            className="pf-u-font-size-sm"
+                                        >
+                                            {values.emailBody.length} / {maxEmailBodyLength}{' '}
+                                            characters
+                                        </Text>
+                                    </TextContent>
+                                </FlexItem>
+                                <FlexItem>
+                                    <Button
+                                        className="pf-u-mt-sm"
+                                        variant="link"
+                                        isInline
+                                        isSmall
+                                        onClick={() => setFieldValue('emailBody', defaultEmailBody)}
+                                    >
+                                        Restore default
+                                    </Button>
+                                </FlexItem>
+                            </Flex>
+                        </FormGroup>
+                    </Form>
+                </Tab>
+                <Tab
+                    eventKey={1}
+                    title={<TabTitleText>Preview</TabTitleText>}
+                    isDisabled={isApplyDisabled}
+                >
+                    <Flex
+                        className="pf-u-py-lg"
+                        spaceItems={{ default: 'spaceItemsMd' }}
+                        direction={{ default: 'column' }}
+                    >
+                        <FlexItem>
+                            <TextContent>
+                                <Text component={TextVariants.small}>
+                                    This preview displays modifications to the email subject and
+                                    body only. Data shown in the report parameters are sample data
+                                    meant solely for illustration. For any actual data, please check
+                                    the email attachment in the real report.
+                                </Text>
+                            </TextContent>
+                        </FlexItem>
+                        <FlexItem>
+                            <Card isFlat>
+                                <CardTitle>{values.emailSubject}</CardTitle>
+                                <CardBody>{values.emailBody}</CardBody>
+                                <CardFooter>
+                                    <div>
+                                        <div style={{ padding: '0 0 10px 0' }}>
+                                            <span
+                                                style={{ fontWeight: 'bold', marginRight: '10px' }}
+                                            >
```

Comment:
I'm assuming you are going old-school with the styling here to mimic the lowest common denominator markup required for HTML email.

Reply:
Yea, wanted to make it easy to copy-paste for them if we end up changing anything in this section in the future

Reply:
If this is going to be the source of truth for the HTML email markup, is it worth adding a comment here that style props need to be converted from an object to a plain string?